.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   build:
     working_directory: ~/project/sam-design-system
     docker:
-      - image: circleci/node:14.18.2-bullseye-browsers
+       - image: circleci/node:16.13-browsers
```

Comment:
If MFEs are onto node 18, any reason not to update this image to 18?

Reply:
When I used 18.17 the circle ci throwing so used 16.